### PR TITLE
Update Readme for the new fork

### PR DESCRIPTION
**Pull request by Darin755**
I went though and basically rewrote the readme in order to make it reflect the forking of the original project.  The old readme can be found at OLDREADME.md. (I was going to use READMEmd.old but github didn't like that for some reason)

### CODE REVIEW

**Comment by bqv** [comment]
Looking good. Glad you did this, I was rather hoping someone would

**Comment by zachatrocity** [comment]
Wow thank you for the contribution! I had some changes to the readme in my current PR where i'm getting the build pipeline going again. Once that is merged would you be willing to check conflicts here?

**Comment by Darin755** [comment]
Absolutely just let me know when your ready to merge

**Comment by zachatrocity** [comment]
I also have a designer working on a new splash screen and app icon, once that is done we can update the readme with new images.

I also noticed a pretty sucky issue today after updating the app from the old api to the new 0.17.x api the comment api has changed so comments aren't nested. I'm still gonna release today cause i wanna get some APKs out there and start getting people testing. Just FYI

File: OLDREADME.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+[<img src="https://cdn.rawgit.com/steverichey/google-play-badge-svg/master/img/en_get.svg" height="80">](https://play.google.com/store/apps/details?id=com.LemmurOrg.lemmur)
+[<img src="https://raw.githubusercontent.com/andOTP/andOTP/master/assets/badges/get-it-on-github.png" height="80">](https://github.com/LemmurOrg/lemmur/releases/latest)
+
+A mobile client for [Lemmy](https://github.com/LemmyNet/lemmy) - a federated reddit alternative
+
+<a href="https://www.buymeacoffee.com/lemmur" target="_blank"><img src="https://www.buymeacoffee.com/assets/img/custom_images/orange_img.png" alt="Buy Me A Coffee" style="height: 41px !important;width: 174px !important;box-shadow: 0px 3px 2px 0px rgba(190, 190, 190, 0.5) !important;-webkit-box-shadow: 0px 3px 2px 0px rgba(190, 190, 190, 0.5) !important;" ></a>
+
+</div>
+
+- [lemmur](#lemmur)
```
Comment (zachatrocity) [inline OLDREADME.md:26]: @Darin755 I'd like to keep from here down in the readme, it has good info for contributors. Also could we add a link to the contribution guide in the `CONTRIBUTING.md`?

My changes are in so should be good to go after these changes

File: README.md [hunk 2]
```diff
@@ -1,72 +1,31 @@
...
 
-This project has been officially dropped due to lack of interest and political differences. If anyone is interested in continuing developement, feel free to fork it. For any questions you can message [krawieck](https://matrix.to/#/@krawieck:matrix.org) (who was responsible for the flutter app) or [shilangyu](https://matrix.to/#/@shilangyu:matrix.org) (who was responsible for lemmy_api_client).
+# Lemmynade
 
----
+A fork of the original lemmur app we all knew and loved. It was unfortunately abandoned but this project aims to keep the legacy it started.
 
-<div align="center">
+## How to Install
+- Currently there are no binary downloads available as this is just the start of this fork. However, you can reference the old readme [here](OLDREADME.md).
```
Comment (zachatrocity) [inline README.md:8]: There are now binaries to install in the releases tab. F-Droid, google play, and iOS are pending verification.

I updated the readme on master so you can just pull the screenshots and howto into that readme, then you can delete OLDREADME and the old privacy policy

**Comment by zachatrocity** [comment]
This has become stale and readme is already updated. Closing

**Pull request closed without merging**